Doc/howto/free-threading-python.rst

```diff
@@ -0,0 +1,128 @@
+.. _freethreading-python-howto:
+
+**********************************************
+Python experimental support for free threading
+**********************************************
+
+Starting with the 3.13 release, CPython has experimental support for running
+with the :term:`global interpreter lock` (GIL) disabled in a build of Python
+called :term:`free threading`.  This document describes the implications of
+free threading for Python code.  See :ref:`freethreading-extensions-howto` for
+information on how to write C extensions that support the free-threaded build.
+
+.. seealso::
+
+   :pep:`703` – Making the Global Interpreter Lock Optional in CPython for an
+   overall description of free-threaded Python.
+
+
+Installation
+============
+
+Starting with Python 3.13, the official macOS and Windows installers
```

Comment:
We don't provide Linux installers.

Reply:
Right, but we could mention here that people building their own CPython such as on Linux need to use`configure --disable-gil` to create such an installation.

Reply:
Mostly I want to direct people to https://py-free-threading.github.io/installing_cpython/. That's going to be a lot more comprehensive. There are a lot of ways people install Python!

I can make that part of the body of the text instead of a "see also" section